website/versioned_docs/version-5.x/server-side-rendering-overview.md

```diff
@@ -0,0 +1,294 @@
+
+
+**Before deployment of navbar**:
+
+```json
+{
+  "imports": {
+    "@org-name/navbar/": "https://cdn.example.com/navbar/v1/"
+  }
+}
+```
+
+**After deployment of navbar**:
+```json
+{
+  "imports": {
+    "@org-name/navbar/": "https://cdn.example.com/navbar/v2/"
+  }
+}
+```
+
+The import map itself is hosted on the CDN, so that deployments may occur without restarting the web server. An example of this setup is [shown here](https://github.com/isomorphic-microfrontends/root-config/blob/master/server/index-html.js).
+
+### B. HTTP Request
+
+It is also possible to implement the fetching of HTML content and HTTP headers from microfrontends using HTTP requests. In this setup, **each microfrontend must run as a deployed web server**. The root web server (responsible for responding to the browser) makes an HTTP call to each of the microfrontends' web servers. Each microfrontend web server responds with an HTML page as response body, along with its HTTP response headers. The response body is streamed to the root web server so that it can send the bytes as soon as possible to the browser.
+
+In the context of single-spa-layout, this is done with the `renderApplication` function:
+
+```js
+import {
+  constructServerLayout,
+  setResponseHeaders,
+  renderServerResponseBody,
+} from "single-spa-layout/server";
+import http from 'http';
+import fetch from 'node-fetch';
+
+const serverLayout = constructServerLayout({
+  filePath: "server/views/index.html",
+});
+
+http.createServer((req, res) => {
+  const { bodyStream } = renderServerResponseBody(serverLayout, {
+    urlPath: req.path,
+    renderApplication(props) {
+      return fetch(`http://${props.name}`).then(r => {
+        if (r.ok) {
+          // r.body is a Readable stream
+          return r.body;
+        } else {
+          throw Error(`Received http response ${r.status} from microfrontend ${props.name}`);
+        }
+      })
+      return import(`${props.name}/server.js`).then(app => {
+        return app.serverRender(props);
+      });
+    },
+    renderFragment(name) {
+      // not relevant to the docs here
+    }
+  });
+
+  bodyStream.pipe(res);
+}).listen(9000)
+```
+
+## 3. HTTP Response Headers
+
+The HTTP response headers sent to the browser are a combination of default headers and the headers retrieved from each microfrontend. Your [method of fetching microfrontends](#2-fetch) does not change how the final headers are merged and assembled for the browser.
+
+Tailor and TailorX have default methods of merging headers. At the time of this writing, single-spa-layout requires manual merging of the headers.
```

Comment:
This seems to be more of an aside so it might be good to put it into a side comment (though Blockquotes shouldn't be used for this... but that's how they're used almost everywhere with Markdown).

```suggestion
> At the time of this writing, single-spa-layout requires manual merging of the headers; while Tailor and TailorX have default methods of merging headers. 
```